content/docs/usage.md

```diff
@@ -117,13 +117,13 @@ pipelines:
         name: Train model
         script:
           - pip install -r requirements.txt
-          - python train.py
+          - python train.py # generate plot.png
     - step:
         name: Create CML report
         script:
-          - cat metrics.txt > report.md
-          - echo >> report.md
-          - echo '![](./plot.png)' >> report.md
+          # Post reports as comments in Bitbucket
+          - cat metrics.txt >> report.md
+          - echo '![](./plot.png "Confusion Matrix")' >> report.md
```

Comment:
Unnecessarily verbose demonstration of arcane Markdown syntax?

Reply:
Perhaps; what does @iterative/cml think?

Reply:
The same it thought on October 23rd. 😉 Let's ask [the rest of] @iterative/cml.